package.json

```
@@ -26,10 +26,12 @@
   },
   "dependencies": {
     "apollo-server": "2.9.3",
+    "apollo-server-express": "^2.9.7",
```

There are two schools of thought for this case, the first thing is better to use `^` the other not, but in any case until we commit and package the `package-lock.json` non of this really matter because the version will still remain fixed by it, for this reason, in my opinion, is better to remove the `^` because is misleading and is not doing anything.
```suggestion
    "apollo-server-express": "2.9.7",
```